ui/helpers/displayHelper.js

```diff
@@ -0,0 +1,14 @@
+export const toDateStr = function toDateStr(timestamp) {
+  const dateFormat = {
+    weekday: 'short',
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: false
+  };
+  return new Date(timestamp * 1000).toLocaleString("en-US", dateFormat);
+};
+
+export default toDateStr;
```

Comment:
The default export here is presumably to quieten eslint until we have more in this file. Another way to do the same is:
https://github.com/mozilla/treeherder/blob/b51909d45ad3cc94978026e54614ee0e564c8773/ui/js/constants.js#L1-L2